.dockerignore

```diff
@@ -0,0 +1 @@
+.venv
```

Comment:
ignore more files that are not needed for running the app(like images(result.png for example), readme, dockerfile, etc)